src/components/about/About.js

```diff
@@ -37,10 +36,10 @@ const About = () => {
                 PB2020/police-brutality GitHub Repository
               </a>
               {'. '}
-              This repository accumulates and contextualizes crowdsourced
```

Comment:
I'm re-reading this thing multiple times to be sure: I think this was actually ok as "contextualizes" vs. "contextualize".

Reply:
Yeah I agree. I kept second guessing myself when reading that lol